src/components/shared/frame/Header/index.tsx

```diff
@@ -82,17 +88,30 @@ export const Header = ({ main, user }: { main: Tree[]; user: string[] }) => {
           </div>
         </MainNavigation>
       </header>
-      <div className="mobileHeader">
-        <img src={`${getAssetBase()}/images/logos/cx-short.svg`} alt="logo" />
-      </div>
-      <div
-        onClick={() => dispatch(setAppear({ SEARCH: !visible }))}
-        className="mobile-search-icon"
-        onKeyUp={() => {
-          // do nothing
-        }}
-      >
-        <SearchIcon sx={{ color: '#0f71cb' }} />
+      <div className="mobileNav">
+        <div className="mobileHeader">
+          <img src={`${getAssetBase()}/images/logos/cx-short.svg`} alt="logo" />
+        </div>
+        <div className="mobileHeaderRight">
+          <div
+            onClick={() => dispatch(setAppear({ SEARCH: !visible }))}
+            className="mobile-search-icon"
+            onKeyDown={() => {
+              //ignore
```

Comment:
for uniformity sake let's stay with
`// do nothing`
here, which is also what eslint has as an example for the no-empty-function rule:
https://eslint.org/docs/latest/rules/no-empty-function